packages/visualizations-react/src/components/ChoroplethGeoJson.tsx

```diff
@@ -1,12 +1,12 @@
-import { FC } from 'react';
 import {
-    DataFrame,
-    SvgChoropleth as _SvgChoropleth,
+    ChoroplethGeoJson as _ChoroplethGeoJson,
     ChoroplethOptions,
```

Comment:
Each React wrapper can now return exactly the specific type (`ChoroplethGeoJsonOptions`, `ChoroplethVectorTilesOptions`)